translations/es.ts

```diff
@@ -1101,7 +1101,7 @@ en lugar de la barra de tareas del sistema.</translation>
     <message>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="200"/>
         <source>Your status is changed to Away after set period of inactivity.</source>
-        <translation>Tu estado se cambia a Ausente después del período de inactividad establecido.</translation>
+        <translation>Tu estado cambia a &apos;Ausente&apos; después del período de inactividad establecido.</translation>
```

Comment:
I don't want anybody to be harmed, let alone die!! ;)
No todos mis cambios significan que haya estado mal. Algunas cosas son de gusto. Y normalmente cuando usas una palabra que tiene un significado propio, como lo tiene 'ausente', la pones en ' o ". Aquí 'ausente' no la función de trnasmitir la noción de que algo no esté aquí, pero se refiere a y como palabra representa un estado que marca tu programa.
Para mí, mayúsculas sirven para nombres propios, restaltar algo, títulos, …